tests/scenario/conftest.py

```diff
@@ -54,6 +54,7 @@ def traefik_container(tmp_path):
         layers={"traefik": layer},
         exec_mock={
             ("update-ca-certificates", "--fresh"): ExecOutput(),
+            ("find", "/opt/traefik/juju", "-name", "*.yaml", "-delete"): ExecOutput(),
```

Comment:
Seems like we need a pebble feature request.

Reply:
not sure this is generic enough to be useful to many, but we can try :)

Reply:
https://github.com/canonical/operator/issues/1042